src/widgetastic/browser.py

```diff
@@ -811,14 +816,12 @@ def __eq__(self, other):
         return self._o == other._o and self._browser == other._browser
 
     def elements(
-            self, locator, parent=None, check_visibility=False, check_safe=True,
-            force_check_safe=False):
+            self, locator, parent=None, check_visibility=False, check_safe=True):
```

Comment:
Not seeing any direct use of this kwarg in wt.patternfly or integration_tests so I think its clean to remove.